src/Microsoft.AspNetCore.Blazor/Components/Provider.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Blazor.RenderTree;
+using System;
+
+namespace Microsoft.AspNetCore.Blazor.Components
+{
+    /// <summary>
+    /// A component that provides one or more parameter values to all descendant components.
+    /// </summary>
+    public class Provider : IComponent
+    {
+        private RenderHandle _renderHandle;
+
+        /// <summary>
+        /// The content to which the value should be provided.
+        /// </summary>
+        [Parameter] private RenderFragment ChildContent { get; set; }
+
+        /// <summary>
+        /// The value to be provided.
+        /// </summary>
+        [Parameter] private object Value { get; set; }
```

Comment:
I assume I'll find the answer when I look at subsequent commits, but it looks like based on just this commit that `Value` doesn't go anywhere yet 😆

Reply:
Correct